conan/cli/formatters/graph/graph.py

```diff
@@ -99,6 +100,14 @@ def _render_graph(graph, template, template_folder):
 def format_graph_html(result):
     graph = result["graph"]
     conan_api = result["conan_api"]
+    field_filter = result["field_filter"]
+    if field_filter and "requires" not in field_filter:
+        field_filter.append("requires")
+    package_filter = result["package_filter"]
+    serial = graph.serialize()
+    # TODO: This is not used, it is necessary to update the renderings to use the serialized graph
+    #  instead of the native graph
+    serial = filter_graph(serial, package_filter, field_filter)
```

Comment:
Do we need to save the output of the function if it's not used elsewhere?

Reply:
It is temporary, with the ``TODO``. The idea is that this ``serial`` is the thing that should be fed to the rendering templates, and not the graph object.

Reply:
Ohh now the PR comment makes sense, I didn't quite understand what it was in reference to, ty for the clarification!